cproton.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+ #!/usr/bin/env bash
```

Comment:
I think you'll want to remove this space so the shebang works properly.
```suggestion
#!/usr/bin/env bash
```

Reply:
Thanks, fixed it